infraohjelmointi_api/management/commands/locationimporter.py

```diff
@@ -17,16 +20,24 @@ def add_arguments(self, parser):
             ),
             default="",
         )
+        parser.add_argument(
+            "--eri-kaupunginosia",
```

Comment:
I was wondering, should we keep script arguments in English. 🤔

Reply:
Generally speaking, yes. But in this case the location option is called "Eri kaupunginosia" so I think it makes sense and is easier to connect to the option shown in the UI. Also, this script will be deleted after the change is done. It's used only once to add the new option to the database